README.md

```diff
@@ -311,3 +313,14 @@ cargo build --no-default-features --features freebsd
 # Running the tool
 cargo run --no-default-features --features freebsd
 ```
+
+Build UEFI tool
+
+```
+# Build just tool
+gmake -C framework_uefi build/x86_64-unknown-uefi/boot.efi
+
+# Build QEMU image
+# TODO: Does not work yet, need a replacement for GNU parted
```

Comment:
@aokblast is there a scripted way to format a disk image on FreeBSD?
On Linux I used GNU parted.

Reply:
Hello, we have ``gpart`` which is the abbriviate of ``geom part`` that enable you to create and add partition. I found a [post](https://serverfault.com/questions/198925/how-to-format-a-partition-under-freebsd) on stack exchange that have examples about this. Do I answer your question?

Reply:
Oh, you are talking about image. We have mkimg which can be used as following pattern (stolen from my previous commit :) )
```
mkimg -s mbr \
    -b ${BASEBITSDIR}/boot/mbr \
    -p efi:=${espfilename} \
    -p freebsd:-"mkimg -s bsd -b ${BASEBITSDIR}/boot/boot -p freebsd-ufs:=${2}.part" \
    -p fat16b:=${2}.config\
```

Reply:
oh! That's cool. That looks super easy!

Reply:
Haven't tried, but might be as simple as this:

```
$(BUILD)/boot.img: $(BUILD)/boot.efi
  mkimg -s gpt \
    -p efi:=$<
```

But how can I use different make rules on FreeBSD :thinking:

Reply:
What is the make rules you mentioned? Could you give me some detailed example?

Reply:
Oops, I forget some code segment. You have to create a partition image by mkfs than put file into it by directory. After that, you can use mkimg to create a full disk image. Here is a [patch](https://reviews.freebsd.org/D38838) use this feature. You can take a release/amd64/make-memstick.sh as example. Hope I answered your question.

Reply:
Done! Now we just need to make sure it uses different commands on linux vs bsd